legacy/scripts/exec-generate-insights-configmap.sh

```diff
@@ -107,6 +107,18 @@ processSbom() {
         annotate configmap ${SBOM_CONFIGMAP} \
         lagoon.sh/branch=${BRANCH}
     fi
+    # Support custom Depdency Track integration.
+    local endpoint=$(featureFlag INSIGHTS_DEPENDENCY_TRACK_API_ENDPOINT)
+    local dtWarn
+    if [ ! -z "$endpoint" ]; then
+      if [ ! -z "$(featureFlag INSIGHTS_DEPENDENCY_TRACK_API_KEY)" ]; then
+        kubectl -n ${NAMESPACE} \
+          annotate configmap ${SBOM_CONFIGMAP} \
+          insights.lagoon.sh/dependency_track_custom_endpoint=${endpoint}
```

Comment:
annotations typically use `-` instead of `_`, see the common annotations that kubernetes uses https://kubernetes.io/docs/reference/labels-annotations-taints/

This was described in the [proposal](https://github.com/uselagoon/lagoon/issues/3597) to update other labels/annotations that lagoon uses in the future